packages/create-nx-workspace/bin/create-nx-workspace.ts

```diff
@@ -605,6 +624,75 @@ async function determineStyle(
   return Promise.resolve(parsedArgs.style);
 }
 
+async function determineNxCloud(
+  parsedArgs: yargs.Arguments<Arguments>
+): Promise<boolean> {
+  if (parsedArgs.nxCloud === undefined) {
+    return enquirer
+      .prompt([
+        {
+          name: 'NxCloud',
+          message: `Use Nx Cloud? (It's free and doesn't require registration.)`,
+          type: 'select',
+          choices: [
+            {
+              name: 'Yes',
+              hint: 'Faster builds, run details, GitHub integration. Learn more at https://nx.app',
+            },
+
+            {
+              name: 'No',
+            },
+          ],
+          initial: 'Yes' as any,
+        },
+      ])
+      .then((a: { NxCloud: 'Yes' | 'No' }) => a.NxCloud === 'Yes');
+  } else {
+    return parsedArgs.nxCloud;
+  }
+}
+
+async function determineCI(
+  parsedArgs: yargs.Arguments<Arguments>,
+  nxCloud: boolean
+): Promise<string[]> {
+  if (!nxCloud) {
+    if (parsedArgs.ci && parsedArgs.ci.length > 0) {
+      output.warn({
+        title: 'Invalid CI value',
+        bodyLines: [
+          `CI option only works when Nx Cloud is enabled.`,
+          `The value provided will be ignored.`,
+        ],
+      });
+    }
+    return [];
+  }
+
+  if (parsedArgs.ci) {
+    return parsedArgs.ci;
+  }
+
+  if (parsedArgs.allPrompts) {
+    return enquirer
+      .prompt([
+        {
+          name: 'CI',
+          message: `Autogenerate CI workflow file (multi-select)?`,
+          type: 'multiselect',
+          choices: [
+            { message: 'GitHub Actions', name: 'github' },
+            { message: 'Circle CI', name: 'circleci' },
+            { message: 'Azure DevOps', name: 'azure' },
+          ],
```

Comment:
Is there a way to select none?

Reply:
Yes, it's a multi-select so you can select 0..3

Reply:
That is also the default value